data.sql

```diff
@@ -40,3 +40,36 @@ UPDATE animals SET owner_id = CASE
     WHEN name IN ('Charmander', 'Squirtle', 'Blossom') THEN (SELECT id FROM owners WHERE full_name = 'Melody Pond')
     WHEN name IN ('Angemon', 'Boarmon') THEN (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
 END;
+INSERT INTO vets (name, age, date_of_graduation) VALUES
+    ('Vet William Tatcher', 45, '2000-04-23'),
+    ('Vet Maisy Smith', 26, '2019-01-17'),
+    ('Vet Stephanie Mendez', 64, '1981-05-04'),
+    ('Vet Jack Harkness', 38, '2008-06-08');
```

Comment:
[OPTIONAL]
- [ ] Please note that since we are inserting into the vets' table, there is no need to prefix each name with **Vet** Keyword.